python/cudf/cudf/core/column/lists.py

```diff
@@ -655,6 +650,11 @@ def sort_values(
         1         [2.0, 8.0, 8.0]
         2              [1.0, 2.0]
         dtype: list
+
+        .. pandas-compat::
+            **list.ListMethods.sort_values**
+
+            * Not supporting: `inplace`, `kind`
```

Comment:
Let's reword this to
```suggestion
            The ``inplace`` and ``kind`` arguments are currently not supported.
```